src/xcode/ENA/ENA/Source/Scenes/Events/Cells/EventTableViewCell.swift

```diff
@@ -88,6 +88,9 @@ class EventTableViewCell: UITableViewCell {
 		durationStackView.isHidden = cellModel.isDurationStackViewHidden
 
 		titleLabel.text = cellModel.title
+		titleLabel.accessibilityLabel = cellModel.titlePrefix
+		titleLabel.accessibilityIdentifier = cellModel.titlePrefix
```

Comment:
You don't need to set the accessibilityIdentifier here at all, that's what I was trying to say with my previous suggestion. The label is all VoiceOver needs. The identifier is not exposed to the user.

Reply:
Yeah just noticed that.